images/input_output.svg

```diff
@@ -1,21 +1,21 @@
+@startuml
+start
+-> input;
+:Jamal Processing;
+-> output;
+: $input;
+:Defer macro evaluation;
+-> output;
+: $output;
+stop
+@enduml
 
-PlantUML version 1.2021.9(Sun Jul 25 12:13:56 CEST 2021)
+PlantUML version 1.2022.4(Sat Apr 09 15:29:17 CEST 2022)
 (GPL source distribution)
 Java Runtime: OpenJDK Runtime Environment
 JVM: OpenJDK 64-Bit Server VM
 Default Encoding: UTF-8
 Language: en
-Country: GB
+Country: HU
```

Comment:
??

Reply:
Could you remove all SVG files from the pull request? They are generated and you have not changed the source that they were generated. At the same time I feel like that you may have a different version fo PlantUML, and thus they may seem different? Even broken some of them.